src/lib.rs

```diff
@@ -823,6 +833,39 @@ impl Config {
         return dst;
     }
 
+    fn getenv_os(&self, v: &str) -> Option<OsString> {
+        let mut cache = self.env_cache.lock().unwrap();
+        if let Some(val) = cache.get(v) {
+            return val.clone();
+        }
+        let r = env::var_os(v);
+        println!("{} = {:?}", v, r);
+        cache.insert(v.to_string(), r.clone());
+        r
+    }
+
+    /// Gets a target-specific environment variable.
+    fn getenv_target_os(&self, var_base: &str) -> Option<OsString> {
+        let target = match self.target.clone() {
+            Some(t) => t,
+            None => {
+                let mut t = getenv_unwrap("TARGET");
+                if t.ends_with("-darwin") && self.uses_cxx11 {
+                    t = t + "11"
```

Comment:
How come this is here? (I'd expect this to not have a special case here)

Reply:
This is a copy paste from https://github.com/alexcrichton/cmake-rs/pull/120/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R419-R428.

Reply:
Actually yeah this probably should be removed. Fixed in latest push.